### PR TITLE
Homework 1 - Sam Sustacek

### PR DESCRIPTION
**Pull request by samsustacek**
This final product is the result of probably 3 iterations, so this was a pretty challenging process. However, the actual programming makes a lot of sense. It's more using Github, R and just orienting myself to the tools of the class that is tough.
I used the instructions on Prof. Soltoff's website and on Github to go about forking, downloading, and editing the .md file from the Class Repository. This was a fairly simple step, but I had a bit of an issue getting Github to work properly in R.
After that, I simply followed the guides to create my biography. It was fairly simple to use bold, italics, headers, lists, links, and even images.
The commit/push/pull process was quite difficult, as I got a variety of errors throughout this process. I could never get the push to work, but with Mint's help, I finally figured out I needed to update R and Re-login to github on R in order to fix it. 
I think once I am more comfortable with the lingo and tools of programming, it'll become easier to problem solve on my own, but I'm excited because this was a fun challenge.

### CODE REVIEW

**Review by ericgao29 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Your biography is pretty interesting! And the way of presenting info is also well organized.
* Maybe next time you can try to insert links into some words by using: `[some words](links)`.
E.g. If you don't believe me, I brought [proof](https://athletics.uchicago.edu/sports/mbkb/2017-18/bios/sustacek_sam_lp0o)!

File: Hw01-Official Submission.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# hw01
```
Comment (bensoltoff) [inline Hw01-Official Submission.md:1]: You have three different copies of the README in your repo. You don't have to create duplicates. Remember that if you use Git correctly, you are tracking all changes you make to the document. You don't need separate copies as backups

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# Sam Sustacek's Biography
+My name is Sam Sustacek, as you can tell by the title, and I am a **4th year** at the Univeristy of Chicago. Originally, I'm from Minneapolis, MN, but I haven't lived there in *3 years*, because I spend all of my time in Chicago.
```
Comment (bensoltoff) [inline README.md:2]: Best practice is to leave a line break between headers and text blocks. Avoids some problems with formatting when the document is rendered

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+## Activities and Interests
+I've participated in quite a few things since I've been at the University of Chicago. These include:
+* Basketball
+  * Varsity in High School and College
+* Painting
+* Writing Club
+* Cooking
+  * Lots of tacos and curry
+
+I had to stop playing basketball after 2 years in college beacuse I had **2 surgeries in 6 months**, but I swear I really did play. If you don't believe me, I brought proof! If you click on this link (https://athletics.uchicago.edu/sports/mbkb/2017-18/bios/sustacek_sam_lp0o) you can see my profile page! I've also included a picture of me playing as proof!
```
Comment (bensoltoff) [inline README.md:13]: Alternative formatting:

```
[If you click on this link](https://athletics.uchicago.edu/sports/mbkb/2017-18/bios/sustacek_sam_lp0o)
```

[If you click on this link](https://athletics.uchicago.edu/sports/mbkb/2017-18/bios/sustacek_sam_lp0o)

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

**Pull request closed without merging**